src/v/redpanda/application.cc

```diff
@@ -1753,30 +1755,27 @@ void application::wire_up_redpanda_services(
                         return cfg.name == ep.name;
                     });
                   // if tls is configured for this endpoint build reloadable
-                  // credentails
+                  // credentials
                   if (it != tls_config.end()) {
                       syschecks::systemd_message(
                         "Building TLS credentials for kafka")
                         .get();
-                      auto kafka_builder
-                        = it->config.get_credentials_builder().get0();
-                      credentails
-                        = kafka_builder
-                            ? kafka_builder
-                                ->build_reloadable_server_credentials(
-                                  [this, name = it->name](
-                                    const std::unordered_set<ss::sstring>&
-                                      updated,
-                                    const std::exception_ptr& eptr) {
-                                      rpc::log_certificate_reload_event(
-                                        _log, "Kafka RPC TLS", updated, eptr);
-                                  })
-                                .get0()
-                            : nullptr;
+                      credentials
+                        = net::build_reloadable_server_credentials_with_probe(
+                            it->config,
+                            "kafka",
+                            it->name,
+                            [this](
+                              const std::unordered_set<ss::sstring>& updated,
+                              const std::exception_ptr& eptr) {
+                                rpc::log_certificate_reload_event(
+                                  _log, "Kafka RPC TLS", updated, eptr);
+                            })
+                            .get0();
```

Comment:
nitpick: `get0` was deprecated a long time ago
```suggestion
                            .get();
```

Reply:
Not sure how that crept in... 👍

Reply:
> Not sure how that crept in... 👍 

It was already there.

Reply:
>not sure how that crept in

It was already there. `application.cc` is full of 'em.